baseapp/abci.go

```diff
@@ -252,18 +251,18 @@ func (app *BaseApp) CheckTx(ctx context.Context, req *abci.RequestCheckTx) (*abc
 
 // DeliverTxBatch executes multiple txs
 func (app *BaseApp) DeliverTxBatch(ctx sdk.Context, req sdk.DeliverTxBatchRequest) (res sdk.DeliverTxBatchResponse) {
-	scheduler := tasks.NewScheduler(app.concurrencyWorkers, app.TracingInfo, app.DeliverTx)
-	// This will basically no-op the actual prefill if the metadata for the txs is empty
-
-	// process all txs, this will also initializes the MVS if prefill estimates was disabled
-	txRes, err := scheduler.ProcessAll(ctx, req.TxEntries)
-	if err != nil {
-		// TODO: handle error
-	}
-
 	responses := make([]*sdk.DeliverTxResult, 0, len(req.TxEntries))
```

Comment:
is it safe to create a slice with capacity 0? i think its probably fine but just checking

Reply:
yes it's safe (just didn't want it to be nil for marshaling reasons, which would be idiomatic golang)